Server/ReasnAPI/ReasnAPI/Controllers/EventsController.cs

```diff
@@ -262,36 +274,22 @@ public IActionResult AddEventComment([FromRoute] string slug, [FromBody]CommentR
 
     [HttpGet]
     [Authorize(Roles = "Admin, Organizer")]
-    [Route("{slug}/parameters")]
-    [ProducesResponseType<List<ParameterDto>>(StatusCodes.Status200OK)]
-    public IActionResult GetEventsParameters([FromRoute] string slug)
+    [Route("/parameters")]
+    [ProducesResponseType<List<string>>(StatusCodes.Status200OK)]
+    public IActionResult GetEventsParameters()
     {
-        var @event = eventService.GetEventBySlug(slug);
-        var user = userService.GetCurrentUser();
-        if (user.Role != UserRole.Admin && @event.OrganizerId != user.Id)
-        {
-            Forbid();
-        }
-
-        var parameters = @event.Parameters.ToDtoList();
+        var parameters = parameterService.GetAllParameterKeys().ToList();
         return Ok(parameters);
     }
 
     [HttpGet]
-    [Authorize(Roles = "Admin, Organizer")]
-    [Route("{slug}/tags")]
+    [Authorize(Roles = "Admin, Organizer\"")]
+    [Route("/tags")]
     [ProducesResponseType<List<TagDto>>(StatusCodes.Status200OK)]
-    public IActionResult GetEventsTags([FromRoute] string slug)
+    public IActionResult GetEventsTags()
     {
-        var user = userService.GetCurrentUser();
-        var @event = eventService.GetEventBySlug(slug);
-
-        if (user.Role != UserRole.Admin && @event.OrganizerId != user.Id)
-        {
-            Forbid();
-        }
-
-        var tags = @event.Tags.ToDtoList();
+       
+        var tags = tagService.GetAllTags().ToList();
```

Comment:
This returns list of string, right?

Reply:
Oh I see - we can do like in params, we only need the text, less nesting the better. To not do result.data.tags.map... name.value... and do just result.map etc

Reply:
i will return string list of tag names